### ISSUE TITLE
Albums have a limit of 99 songs

### ISSUE THREAD

**Issue by Aese0** [25huizengek1/ViTune#669]
### Steps to reproduce the bug

Find any album with 100+ songs.
For example, search for "极限委托" in albums.
Open the album.

### Expected behavior

Album displays all of its songs.
All songs in the album are listed in the queue.

### Actual behavior

Only 99 songs are accessible in the album 
Queue shows only the first 99 songs from the album.
If you play a song from the latter half of the album (greater than 99), they are not placed next to the other songs, that are part of the album, in 'Songs' - saved Songs tab.

### Screenshots/Screen recordings

![Screenshot_2024-12-27-12-35-47-762_app vitune android](https://github.com/user-attachments/assets/02537ac7-7a6c-4f34-8d68-0093e7027a60)
![Screenshot_2024-12-27-12-35-51-554_app vitune android](https://github.com/user-attachments/assets/fc53fad0-2ea5-43e3-a941-d191019f3542)
![Screenshot_2024-12-27-12-27-27-283_com google android apps youtube music](https://github.com/user-attachments/assets/a1fabbdc-5b18-4811-a8cd-42e36a640b8c)

### ViTune version

v1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 15

### Device info

Xiaomi 13T Pro, HyperOS 2

### Checklist

- [X] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by 25huizengek1** [comment]
@th3y That is true. I haven't even looked into this but this is indeed the fix. During development, the possibility of albums behaving the same as playlists on YouTube wasn't taken into consideration, for some reason.
